back/src/bsda/resolvers/mutations/__tests__/update.integration.ts

```diff
@@ -2489,5 +2493,375 @@ describe("Mutation.updateBsda", () => {
         );
       });
     });
+
+    it("[bug prod] if user updates the BSDA by adding a nextDestination but does not modify the CAP - should not send mail", async () => {
```

Comment:
```suggestion
    it("[bug tra-15665] if user updates the BSDA by adding a nextDestination but does not modify the CAP - should not send mail", async () => {
```

Je trouve ça cool de faire le lien avec le ticket dans ce genre de cas pour avoir une meilleure compréhension du problème.